src/main/java/br/com/meli/exercises/controller/AgeController.java

```diff
@@ -0,0 +1,18 @@
+package br.com.meli.exercises.controller;
+
+import br.com.meli.exercises.service.impl.AgeServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AgeController {
+    @Autowired
+    public AgeServiceImpl ageService = new AgeServiceImpl();
```

Comment:
Não acessa o Impl, e sim a interface.
Outra, usa injeção por construtor.

```suggestion
    private final AgeService ageService;
    
    public AgeController(AgeService ageService) {
          this.ageService = ageService;
    };
```